New custom touch controls.

Kode-Z opened the pull request:
Added custom touch controls for DOOM + DOOM II (last years version), Conker: Live and Reloaded (Remote Play) and updated the controls for South of Midnight to have segmented pull buttons for easier combat/traversal!

![image](https://github.com/user-attachments/assets/2082398e-f828-43d7-8de3-5e1e58257d1a)
![image](https://github.com/user-attachments/assets/ecc16364-aa7f-423c-919d-77db23e7ebb5)
![image](https://github.com/user-attachments/assets/5c3a00be-6b97-4f42-8a14-aaece98d8e10)

Kode-Z committed:
Add files via upload

Kode-Z committed:
Add files via upload

Kode-Z committed:
Update 1278031123.json

Kode-Z committed:
Merge branch 'redphx:gh-pages' into gh-pages

Kode-Z committed:
Update south-of-midnight.json

Added segmented pull buttons to enhance combat/traversal gameplay!

Kode-Z committed:
Add files via upload

Kode-Z committed:
Add files via upload

Kode-Z committed:
Add files via upload

Kode-Z committed:
Add files via upload

redphx commented:
Can you adjust  the opacity of these button's colors? I think it's a little distracting.

![image](https://github.com/user-attachments/assets/b87d020c-3298-42d7-8f4c-a3c109dfc778)

Kode-Z commented:
Where would I put the "opacity" line in? Can't seem to get it working!

Kode-Z commented:
I've figured it out! Will update in a moment.

Kode-Z committed:
Update south-of-midnight.json

Kode-Z committed:
Update south-of-midnight.json

Kode-Z commented:
@redphx  I've updated the file.

redphx committed:
south-of-midnight: minor adjustments

- Change segmented button's opacity from 0.3 to 0.5
- Add "exclusive" type for d-pad

redphx commented:
I made some adjustments. Thanks for the contributions.

Kode-Z commented:
Thanks!
